Dockerfile

```diff
@@ -18,5 +20,8 @@ RUN apk --update add $PACKAGES && \
 
 RUN pip install vim-vint==0.3.15
 
-RUN git clone https://github.com/junegunn/vader.vim vader && \
-    cd vader && git checkout c6243dd81c98350df4dec608fa972df98fa2a3af
+RUN git clone https://github.com/junegunn/vader.vim vader
```

Comment:
I recommend still targeting a specific commit or tag so it doesn't change later and break running some tests. Use the same commit ID in `.appveyor.yml` so the Windows tests run the same version.

Reply:
Done.  I am also trying to migrate appveyor to Github actions. At that point I may as well update vader on both sides.